apps/omg/lib/omg/output.ex

```diff
@@ -60,14 +67,14 @@ defmodule OMG.Output do
 
   # TODO(achiurizo)
   # remove the validation here and port the error tuple response handling into ex_plasma.
-  defp validate_data([raw_type, [owner, currency, amount]]) do
+  defp clean_and_validate_data([raw_type, [owner, currency, amount]]) do
     with {:ok, type} <- RawData.parse_uint256(raw_type),
          {:ok, _} <- valid_output_type?(type),
-         {:ok, _} <- RawData.parse_address(owner),
+         {:ok, owner1} <- RawData.parse_address(owner),
```

Comment:
what do you think about naming this variable and vars below like `parsed_owner` ?